xapian-glib/xapian-query.cc

```diff
@@ -146,6 +146,12 @@ xapian_query_class_init (XapianQueryClass *klass)
   gobject_class->finalize = xapian_query_finalize;
 }
 
+XapianQuery *
```

Comment:
you will need a gtk-doc stanza as well:

```
/**
 * xapian_query_new_match_all:
 *
 * ...
 *
 * Returns: (transfer full): the newly created #XapianQuery instance
 *
 * Since: 1.2
 */
```